### ISSUE TITLE
new beta 4.1 version inoperable

### ISSUE THREAD

**Issue by ClaireCJS** [huaruitech/Hitomi-Downloader#1]
Latest version (this weeks' update) is inoperably broken. First time in years I will ever have to downgrade to continue downloading.

All my jobs are stuck:
![image](https://github.com/user-attachments/assets/53c1397f-30cd-4cef-a1d2-fd3451db873f)

Something changed with this version. There's no way to stop a job.

And if i restart, it doesn't allow a restart because it's already going!

Except it's not. Not a single byte will be downloaded. This persists beyond reboots, even.

Basically, if enough jobs hang... You can't download anymore. Can't stop the jobs. Can't restart. Can't stop.

[Separately, a lot of images are the wrong extension when downloaded. This breaks a lot of compliance checks.]
